src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java

```diff
@@ -0,0 +1,55 @@
+package widecat.meteorcrashaddon.modules;
+
+import widecat.meteorcrashaddon.CrashAddon;
+import meteordevelopment.meteorclient.events.world.TickEvent;
+import meteordevelopment.meteorclient.settings.SettingGroup;
+import meteordevelopment.meteorclient.settings.*;
+import meteordevelopment.meteorclient.systems.modules.Module;
+import meteordevelopment.orbit.EventHandler;
+import net.minecraft.network.packet.c2s.play.PlayerInteractBlockC2SPacket;
+import net.minecraft.util.Hand;
+import net.minecraft.util.hit.BlockHitResult;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Direction;
+import net.minecraft.util.math.Vec3d;
+import java.util.Objects;
+import java.util.Random;
+
+public class NoComCrash extends Module{
+    final Random      r       = new Random();
+    int i = 0;
+    Vec3d pickRandomPos() {
+        int x = this.r.nextInt(0xFFFFFF);
+        int y = 255;
+        int z = this.r.nextInt(0xFFFFFF);
+        return new Vec3d(x, y, z);
+    }
+    private final SettingGroup sgGeneral = settings.getDefaultGroup();
+    private final Setting<Integer> amount = sgGeneral.add(new IntSetting.Builder()
+        .name("amount")
+        .description("How many packets to send to the server per tick.")
+        .defaultValue(15)
+        .min(1)
+        .sliderMax(100)
+        .build()
+    );
+
+    private final Setting<Boolean> autoDisable = sgGeneral.add(new BoolSetting.Builder()
+        .name("auto-disable")
+        .description("Disables module on kick.")
```

Comment:
I'd rather write "Disable the module when disconnected" or smth

Reply:
i would remove this setting entirely because it is not being used at all

Reply:
:O didn't even notice that LOL